app/views/_mobile-service-instance-row.html

```diff
@@ -55,14 +55,16 @@ <h4 class="component-label section-label">Details</h4>
               </dd>
             </dl>
           </div>
-          <div class="col-md-6" ng-if="row.serviceType === 'ups'">
-            <h4 class="component-label section-label">Variants</h4>
-            <dl class="dl-horizontal left">
-              <dt ng-repeat-start="variant in row.extendedAnnotations.variants">{{variant.typeLabel}}</dt>
-              <dd ng-repeat-end>
-                <a href="{{variant.url}}">{{ variant.id }}</a>
-              </dd>
-            </dl>
+          <div class="col-md-6" ng-if="(row.extendedAnnotations | size)">
+            <div ng-if="row.serviceType === 'ups'">
```

Comment:
@sedroche  don't think we needed the `class="col-md-12"` here as the child should automatically fill up the parent

Reply:
:+1:

Reply:
yep :+1: